src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
+    <Description>Provides DataFactory management capabilities for Microsoft Azure.</Description>
+    <VersionPrefix>4.13.0</VersionPrefix>
```

Comment:
@nicolaszhang this is going to cause even a bigger confusion, if you release a new nuget package with the same exact version, how will your customer know that .Extended is the new version for Microsoft.Azure.Management.DataFactories nuget?